### PR TITLE
Add Figure releases to announcements 

### PR DESCRIPTION
**Pull request by hflynn**
Copies the Figure release announcements that previously only lived on figure.openmicroscopy.org across to the Announcements section of the website

### CODE REVIEW

File: _posts/2014-10-07-figure-1-0-0.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Release of OMERO.figure 1.0.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.figure 1.0.0.
+---
+
+We are pleased to announce release of OMERO.figure 1.0.0.
```
Comment (jburel) [inline _posts/2014-10-07-figure-1-0-0.md]: the release

File: _posts/2015-04-20-figure-1-1-0.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Release of OMERO.figure 1.1.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.figure 1.1.0.
+---
+
+We are pleased to announce release of OMERO.figure 1.1.0.
```
Comment (jburel) [inline _posts/2015-04-20-figure-1-1-0.md]: the release

File: _posts/2015-05-05-figure-1-1-1.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Release of OMERO.figure 1.1.1
+intro-blurb: The OME team is pleased to announce the release of OMERO.figure 1.1.1.
+---
+
+We are pleased to announce release of OMERO.figure 1.1.1 which
```
Comment (jburel) [inline _posts/2015-05-05-figure-1-1-1.md]: the release

**Comment by hflynn** [comment]
Ah, now find Figure 3.0.0 onwards are already here! I wasn't going to bother listing 2.0.1 since it's just a PyPI fix, if that's okay @jburel ? So this is good for final review now, I'll list it for Monday if you don't have time to look at it today.

File: _posts/2015-04-02-figure-1-0-1.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+title: Release of OMERO.figure 1.0.1
+intro-blurb: The OME team is pleased to announce the release of OMERO.figure 1.0.1.
+---
+This release allows OMERO.figure to be used with the latest 5.1.0 OMERO release.
+This release adds no new features or other bug fixes to OMERO.figure.
+
+However, a major release of OMERO.figure with several new features and bug fixes
+should be available in 2 or 3 weeks.
+
+Grab the release from the OMERO.figure [1.1.1 download page](https://downloads.openmicroscopy.org/figure/1.0.1/).
```
Comment (mtbc) [inline _posts/2015-04-02-figure-1-0-1.md]: Maybe 1.0.1?

File: _posts/2016-10-26-figure-2-0-0.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Release of OMERO.figure 2.0.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.figure 2.0.0.
+---
+
+We are pleased to announce the release of OMERO.figure 2.0.0. This release
+features a complete re-organization of the code and the repository moving to
```
Comment (mtbc) [inline _posts/2016-10-26-figure-2-0-0.md]: I'd omit the hyphen.

**Comment by jburel** [comment]
@hflynn ignore 2.0.1.

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/announcements/ all update now, ready for you to review @will-moore - thanks

**Comment by will-moore** [comment]
Do we support Youtube videos in-line?
It can be nice to see them on the page, if the formatting is not too hard to handle?
Or maybe add a link instead?
E.g. releases 1.0.0, 1.1.0, 1.2.0.

I realise we've not been adding links to PRs in the release announcements. I'm not sure why, but maybe this was simply because they were available on the ```figure.openmicroscopy.org``` posts and we mostly pointed users there?
However, if those posts won't be available in future, it would be nice to have the links to PRs here, since this gives a lot more info to those who are interested - and can be useful for our own reference too.

**Comment by hflynn** [comment]
I followed the pattern of release announcements for other bits of the project, where we do not link directly to PRs. There are some PRs listed in the CHANGELOG, I'd keep them there instead.

I'm not sure if we can embed Youtube videos in the markdown pages, I can certainly add the links though.

**Comment by hflynn** [comment]
Of course the videos work embedded, the set up here is the same as where you originally posted them! Added now anyway.

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/2014/10/07/figure-1-0-0.html
https://snoopycrimecop.github.io/www.openmicroscopy.org/2015/04/20/figure-1-1-0.html
https://snoopycrimecop.github.io/www.openmicroscopy.org/2015/11/24/figure-1-2-0.html
updated

**Comment by hflynn** [comment]
This is ready for final sign off now

**Comment by hflynn** [comment]
Is this good to merge now @will-moore ?

**Comment by will-moore** [comment]
Looks great, thanks.

**Pull request merged**